cdk/src/open-data-platform/data-plane/data-import/write-parcels-data.handler.ts

```diff
@@ -4,14 +4,15 @@ import { ParcelsTableRowBuilder } from '../model/parcels-table';
 import { geoJsonHandlerFactory } from './handler-factory';
 
 const SCHEMA = 'schema';
+const CITY = 'toledo';
```

Comment:
A suggestion for a future PR: we could create a handler for each city by iterating through the city enum in `/client/src/model/states/model/geo_data` and passing its value to the hander as an argument.

Reply:
Good idea! I'm not sure you can use client variables here but we could create an identical one in data import stack to use.